### DISCUSSION TITLE
빌드잇에서 생성한 맵을 월드로 가져올 때 매핑한 이미지가 안 보이는 현상

### DISCUSSION THREAD

**Discussion by runtostar** [naverz/zepeto-studio-kor#100] [5. Q&A]
빌드잇에서 생성한 맵을 월드로 가져올 때, 정육면체 블록에 매핑한 이미지가 보이지 않는 현상이 확인됩니다.
분홍색 블록으로만 현시되는데, 해결 방법 좀 알려주세요.

![image](https://user-images.githubusercontent.com/95363110/146630320-36740629-6cba-49de-857a-e82a9a2e1628.png)

**Comment by cheolhui-lee** [comment]
안녕하세요.
커스텀 이미지는 아직 지원이 안되고 있습니다.
급하게 사용하셔야 된다면 새로 머테리얼을 생성하셔서 커스텀 이미지를 적용하여 사용하시면 될 것 같습니다.
